test/extended/util/oc_copy.go

```diff
@@ -49,7 +49,7 @@ func getUserPartOfNickname(clientCfg *restclient.Config) (string, error) {
 		return "", err
 	}
 	userInfo, err := userClient.Users().Get(context.Background(), "~", metav1.GetOptions{})
-	if kerrors.IsNotFound(err) || kerrors.IsForbidden(err) {
+	if kerrors.IsNotFound(err) || kerrors.IsForbidden(err) || kerrors.IsUnauthorized(err) {
```

Comment:
why is this necessary?  This is an indication that an invalid token or expired or misconfigured client-certificate was used.  In either case, the error should be exposed.

Reply:
This was a leftover from the previous version, I removed it.